src/Microsoft.ML.Core/Utilities/Contracts.cs

```diff
@@ -16,7 +16,11 @@
 using System.IO;
 using System.Threading;
 
+#if PRIVATE_CONTRACTS
+namespace Microsoft.ML.Hidden
```

Comment:
You'll need to update [https://github.com/dotnet/machinelearning/blob/65c3c7c14b5da2655cce2c393a5b85546e5091fc/tools-local/Microsoft.ML.InternalCodeAnalyzer/ContractsCheckAnalyzer.cs#L199-L200](https://github.com/dotnet/machinelearning/blob/65c3c7c14b5da2655cce2c393a5b85546e5091fc/tools-local/Microsoft.ML.InternalCodeAnalyzer/ContractsCheckAnalyzer.cs#L199-L200) for this new namespace.

Reply:
Ok. Thanks.

---
In reply to: [236733617](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236733617) [](ancestors = 236733617)